lib/facter/pid.rb

```diff
@@ -0,0 +1,3 @@
+Facter.add(:pid) do
```

Comment:
May not be the best name for a fact. You may want to instead use something very specific to this module.